pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "spdx-tools"
+authors = [{name = "Ahmed H. Ismail", email = "ahm3d.hisham@gmail.com"}]
+maintainers = [
+    {name = "Philippe Ombredanne", email = "pombredanne@gmail.com"},
+    {name = "SPDX group at the Linux Foundation and others"},
+]
+license = {text = "Apache-2.0"}
+description = "SPDX parser and tools."
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+urls = {Homepage = "https://github.com/spdx/tools-python"}
+requires-python = ">=3.6"
+dependencies = ["ply", "rdflib", "click", "pyyaml", "xmltodict"]
+dynamic = ["version"]
```

Comment:
Didn't know about this, pretty cool!
In case anyone else stumbles across this: https://pypi.org/project/setuptools-scm/, look for "Default versioning scheme"